Makefile

```diff
@@ -125,6 +125,12 @@ ko:
 deploy:
 	LDFLAGS="$(SERVER_LDFLAGS)" GIT_HASH=$(GIT_HASH) GIT_VERSION=$(GIT_VERSION) ko apply -f config/
 
+e2e:
+	go test -c -tags=e2e ./tests
+	go test -c -tags=e2e ./pkg/pki/x509
+	go test -c -tags=e2e ./pkg/pki/tuf
+	go test -c -tags=e2e ./pkg/types/rekord
```

Comment:
can this be `./pkg/...`?

Reply:
That was the first thing I tried, and it failed with this.
```
go test -c -tags=e2e ./pkg/...
cannot use -c flag with multiple packages
```

Reply:
Do these need to be compiled into test binaries? Where is this make command actually being used?

Reply:
For sanity check. It is hard to ensure that the code is compiling without these. Anyone who would have to run e2e to ensure it is compiling.  I am using this manually to check if it is compiling. 

I don't want to add another check to CI as it will slow down things. This is to help anyone contributing to making it easier to test things. That was my intention.

Do you have other thoughts?